### ISSUE TITLE
Featured Projects Tweet Thread for Round 14 verified projects

### ISSUE THREAD

**Issue by laurenluz** [Giveth/giveth-planning#742]

**Comment by NikolaCreatrix** [comment]
Thank you @laurenluz 
I will have the first draft ready for you by this Friday. And thank you for all the feedback about how to make this process even smoother. Keeping text in docs and making sure I fit in with the number of characters etc... okay the next one will be better.

**Comment by NikolaCreatrix** [comment]
Hola @laurenluz 

Here is the draft for R14 
https://docs.google.com/document/d/1XMknJFoIzKxtN5BR6myo36ULKj7xFLckEpRkOG6blao/edit?usp=sharing 

I am still practicing my english language so any feedback or correction suggestions are welcome cause your tweets are always fun and captivating. :)

I checked characters in Twitter, should be okay this time.

**Comment by laurenluz** [comment]
Awesome work @NikolaCreatrix - do you maybe want to include one more though? kind of funny that 2/4 are skateboard initiatives.

Left some comments & suggestions

**Comment by NikolaCreatrix** [comment]
Yeah, lot of skateboarding around lately :) thank you, actually there is one more project...

Okay, I added one more, and also edited the RegensUnite.... solved your comments, and applied suggestions, thank you so much @laurenluz 

https://docs.google.com/document/d/1XMknJFoIzKxtN5BR6myo36ULKj7xFLckEpRkOG6blao/edit?usp=sharing

Looks better now?

**Comment by laurenluz** [comment]
Looks great! I queued the draft up in our scheduler, so it's ready to tweet this week. I copy/pasted the final draft in the doc, just so you can have a look. I made some small changes live... for example, by adding in a final tweet that asks people to RT the first one.

GREAT job, Nikola! These threads are soooo helpful, for me, for our twitter & for the projects!!

Will make an issue for next round that you can take on when Round 15 badges are ready to go out :)
